lib/cfndsl/metadata.rb

```diff
@@ -0,0 +1,26 @@
+require 'cfndsl/jsonable'
+
```

Comment:
This file should not be necessary, as `:Metadata` is an attribute on both templates and resources already.